client-app/core/composables/useNavigations.ts

```diff
@@ -158,6 +191,11 @@ export function useNavigations() {
     matchingRouteName.value = value;
   }
 
+  watch(
+    () => menuSchema,
+    (newValue) => (newValue.value ? getTranslatedMenuLink(newValue.value.header.mobile.account) : null),
```

Comment:
There is no any side effect in the watcher. Looks like the code should work without that.